service/kvmconfig/framework.go

```diff
@@ -18,6 +15,10 @@ import (
 	"github.com/giantswarm/randomkeys"
 	apiextensionsclient "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
 	"k8s.io/client-go/kubernetes"
+
+	"github.com/giantswarm/kvm-operator/service/kvmconfig/v2"
+	"github.com/giantswarm/kvm-operator/service/kvmconfig/v3"
+	"github.com/giantswarm/kvm-operator/service/kvmconfig/v3/key"
```

Comment:
Now we see that it is weird to have the init context not versioned yet. Cool thing. Lets iterate from here.